modules/apps/commerce/commerce-pricing-web/src/main/java/com/liferay/commerce/pricing/web/internal/frontend/taglib/servlet/taglib/CPInstancePriceListsScreenNavigationCategory.java

```diff
@@ -37,7 +37,7 @@
 
 	@Override
 	public String getCategoryKey() {
-		return "price-lists";
+		return CPInstanceScreenNavigationConstants.CATEGORY_KEY_PRICE_LISTS;
```

Comment:
Hey @daiazius, rename it to CPInstanceScreenNavigationConstants.CATEGORY_KEY_CP_INSTANCE_PRICE_LISTS, please. Thx